npm-packages/mongo-id/src/id.js

```diff
@@ -0,0 +1,113 @@
+import EJSON from 'ejson';
+import Random from '@reactioncommerce/random';
```

Comment:
In this case we should probably move Random to NPM as well.

Reply:
You mean deprecate it and use reaction commerce version?

Reply:
Yeah, but it would probably make more sense if we move it under `@meteorjs` so that Reaction Commerce doesn't have to maintain it and it is really clear to everyone else what the main purpose it.